packages/dev/core/src/Materials/Node/Blocks/PBR/pbrMetallicRoughnessBlock.ts

```diff
@@ -1116,10 +1116,6 @@ export class PBRMetallicRoughnessBlock extends NodeMaterialBlock {
         state._emitFunctionFromInclude("imageProcessingDeclaration", comments);
         state._emitFunctionFromInclude("imageProcessingFunctions", comments);
 
-        state._emitFunctionFromInclude("shadowsFragmentFunctions", comments, {

```

Comment:
This should still be in without the replace. I guess this is your test issue :-)

Reply:
I didn't find 'vPositionW' in the shadowsFragmentFunctions.fx，so the test issue may not be here. and I find lightBlock.ts should also be fix, because it includes lightFragment.fx

Reply:
oh, I see what you mean. this line code should not be removed